enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -18,8 +27,39 @@ export const SdkContentWrapper = styled.div<{ font: string; baseUrl?: string }>`
   ${({ theme }) => getRootStyle(theme)}
 
   ${({ baseUrl }) => defaultFontFiles({ baseUrl })}
+  ${({ fontFiles }) =>
+    fontFiles?.map(
+      file => css`
+        @font-face {
+          font-family: "Custom";
```

Comment:
Ah, so the `FontFile` type is supposed to represent the same font, but each file is for a different weight and format. That tripped me up a bit while reading as I thought we are mapping multiple font families - I was wondering why we're using `Custom` for each font file. Got it.

In the `FontFile` type looks like we don't have `fontStyle`, so this looks good.